src/ActionModal/components/ActionModal.vue

```diff
@@ -863,7 +852,7 @@ export default {
       },
       /* istanbul ignore next */
       skip() {
-        return !this.address
+        return !this.session.address
```

Comment:
use `address` getter instead

Reply:
wait, why?

Reply:
because we have the getter for the address. So for keeping the code uniform we should use the same patterns everywhere. This will help making the code more readable and understandable for us and others.

Reply:
IMO it is adding unnecessary complexity in this case, in an already super complex component

Reply:
I disagree

Reply:
let's do the refactor in another PR as well